src/main/java/frc/robot/RobotContainer.java

```diff
@@ -40,23 +40,14 @@ private void initializeSubsystems() {
      * initializes all commands
      */
     public void initializeCommands() {
-        Supplier<Double> x, y, rot;
-        if(RobotConstants.DriverConstants.SQUARED_CONTROLLER_DRIVING) {
-            x = () -> Math.pow(driverXbox.getLeftX(), 2);
-            y = () -> Math.pow(driverXbox.getLeftY(), 2);
-            rot = () -> Math.pow(driverXbox.getRightX(), 2);
-        } else {
-            x = driverXbox::getLeftX;
-            y = driverXbox::getLeftY;
-            rot = driverXbox::getRightX;
-        }
-
+        boolean squared = DriverConstants.SQUARED_CONTROLLER_DRIVING;
         driveWithXboxCMD = new SupplierDriveCMD(
                 swerveSS,
-                x,
-                y,
-                rot,
+                squared ? ()->Math.pow(driverXbox.getLeftX(), 2) : driverXbox::getLeftX,
+                squared ? ()->Math.pow(driverXbox.getLeftY(), 2) : driverXbox::getLeftY,
+                squared ? ()->Math.pow(driverXbox.getRightX(), 2) : driverXbox::getRightX,
                 true);
+
```

Comment:
blank line

Reply:
It's there on purpose
Don't we want to set all of the default commands at the end of the function, separate from the initialization?

Reply:
add bind commands